File: android/app/src/main/java/com/sumitrack/android/domain/models/ProductVariant.kt
package com.sumitrack.android.domain.models

import java.time.Instant

data class ProductVariant(
    val id: String,
    val fkTenant: String,
    val fkProduct: String,
    val name: String,
    val createdAt: Instant,
    val updatedAt: Instant,
    val syncStatus: SyncStatus,
)

File: backend/src/Sumitrack.Api/Models/Entities/ProductVariant.cs
namespace Sumitrack.Api.Models.Entities;

public class ProductVariant
{
    public Guid Id { get; set; }
    public Guid FkTenant { get; set; }
    public Guid FkProduct { get; set; }
    public string Name { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string SyncStatus { get; set; } = "synced";
}
